setup.py

```diff
@@ -23,7 +23,7 @@
     name="autoPyTorch",
     version="0.1.0",
     author="AutoML Freiburg",
-    author_email="zimmerl@informatik.uni-freiburg.de",
+    author_email="eddiebergmanhs@gmail.com",
```

Comment:
Shouldn't it be a freiburg email address?

Reply:
I asked for his email address for this purpose and that's what he gave me. I think its fine.